lib/ModuleBuildError.js

```diff
@@ -38,10 +42,10 @@ class ModuleBuildError extends WebpackError {
 			} else if (typeof err.message === "string" && err.message) {
 				message += err.message;
 			} else {
-				message += err;
+				message += String(err);
```

Comment:
Shouldn’t the message be prepended by `:`?

Reply:
This is done in line 21 - 25

Reply:
Right. Looks like it's ready to ship!